source/opt/inst_bindless_check_pass.cpp

```diff
@@ -563,10 +565,31 @@ void InstBindlessCheckPass::GenCheckCode(
     GenDebugStreamWrite(uid2offset_[ref->ref_inst->unique_id()], stage_idx,
                         {error_id, u_index_id, u_length_id}, &builder);
   }
+  // Generate a ConstantNull, converting to uint64 if the type cannot be a null.
```

Comment:
@greg-lunarg & @jeremy-lunarg any comments on this part of the change? It is based on code in [InstBuffAddrCheckPass::GenCheckCode() ](https://github.com/KhronosGroup/SPIRV-Tools/blob/master/source/opt/inst_buff_addr_check_pass.cpp#L101)

Reply:
I have suggested a small efficiency change below. Otherwise this LGTM.